tests/Cache/CacheTaggedCacheTest.php

```diff
@@ -90,7 +90,7 @@ public function testRedisCacheTagsPushStandardKeysCorrectly()
         $conn->shouldReceive('sadd')->once()->with('prefix:bar:standard_ref', 'prefix:'.sha1('foo|bar').':key1');
         $store->shouldReceive('push')->with(sha1('foo|bar').':key1', 'key1:value');
 
-        $redis->put('key1', 'key1:value');
+        $redis->put('key1', 'key1:value', 0);
```

Comment:
I do not understand, if you are trying to test put without passing any minutes why are you changing this test?